kolibri/utils/tests/test_cli_at_import.py

```diff
@@ -0,0 +1,35 @@
+"""
+Tests for `kolibri.utils.cli` module.
```

Comment:
```suggestion
Tests for `kolibri.utils.cli` module.
These tests deliberately omit `@pytest.mark.django_db` from the tests, so that any attempt to access the Django database during the running of these cli methods will result in an error and test failure.
```